src/core/reference/include/ngraph/runtime/reference/proposal.hpp

```diff
@@ -359,6 +358,10 @@ static void proposal_exec(const T* class_probs,
     bool swap_xy = attrs.framework == "tensorflow";
 
     for (unsigned int batch_idx = 0; batch_idx < batch_num; ++batch_idx) {
+        std::fill(is_dead.begin(), is_dead.end(), 0);
```

Comment:
Do we need to have `is_dead` vector here? Looks like we use it only in the nms method. Can we reduce the number of arguments?

Reply:
Good idea, done